docker-compose.yml

```diff
@@ -7,3 +15,66 @@ services:
       dockerfile: Dockerfile.cron.amd64
     env_file: .env
     restart: always
+    networks:
+      - monitor-net
+    labels:
+      org.label-schema.group: "monitoring"
```

Comment:
Dumb question: what's this label for?

Reply:
It can be used for referencing different groups within a custom user defined network in docker. Also available within the container if needed be.